modules/ModuleBuilder/views/view.modulefields.php

```diff
@@ -87,6 +87,9 @@ public function display()
 
             VardefManager::loadVardef($module_name, $objectName, true);
             global $dictionary;
+            if (!isset($studioClass->mbvardefs) || is_null($studioClass->mbvardefs)) {
+                $studioClass->mbvardefs = new stdClass();
```

Comment:
@jack7anderson7

I think we should either:

1. Remove `$studioClass->mbvardefs->vardefs['fields'] = $f;` entirely. As it does not seem to be used in the tpl replated with this view
2. As an alternative, instantiate a `MBVardefs` class instead of `stdClass`

Reply:
Hi @jack7anderson7,

Since we went with the option 1 ). This part of the code doesn't make sense and can also be removed.